### ISSUE TITLE
"none" shown as label for short description

### ISSUE THREAD

**Issue by MrDaar** [magento/magento2#9161]
### Preconditions
1. Magento: 2.1.5
2. PHP: 7

### Steps to reproduce
1. View a product on the frontend.
2. If you have a translation for "none", your translation will show above the text.
3. i.e.:
```
<none in your language>
<short description>
```
```
ingen
This product is great!
```

### Expected result
1. No attribute label if the label is "none"

### Actual result
1. Label "none" (in your language) is shown.

Possible solution:
`vendor/magento/module-catalog/view/frontend/templates/product/view/attribute.phtml line:37`
```
<?php if ($_attributeLabel != 'none'): ?>
```
becomes
```
<?php if ($_attributeLabel != __('none')): ?>
```

**Comment by mattias** [comment]
We still have this even in 2.1.7 and have been waiting for a fix.

Right now we have decided to just not translate "none" until this is fixed.

**Comment by kweij** [comment]
I came here to note the same problem. It's fixed in the 2.2 RC tag:
https://github.com/magento/magento2/blob/2.2.0-RC1.3/app/code/Magento/Catalog/view/frontend/layout/catalog_product_view.xml

**Comment by orlangur** [comment]
Closing in favor of https://github.com/magento/magento2/issues/10738 (it is the newest one but with `G2 Passed` label).
